### ISSUE TITLE
Saving data back to c3d

### ISSUE THREAD

**Issue by gabrielalsantos** [u0078867/ulema-ul-analyzer#23]
Hello! Happy New Year!
I am running data, but new error appeared.  For the set recovery path and set proc. data path, do I need to select the c3d folder containing the folders of the subjects? Because, I did this, and appeared the following message: “Message: reference to non-existent field ‘SUBJECTS’. Name: saveProcDatatoC3D. Line 89. What Am I doing wrong?

Thank you!

Gabriela.

**Comment by u0078867** [comment]
Hi,

for the *recovery path*, yes, you have to select the folder containing the folders having subject names. For the *processed data path*, you can set it where you want, better not as the same for the recovery data path.

**Comment by gabrielalsantos** [comment]
Hi,
For the processed data path, I selected other folders that is not same of the recovery data path. But, the same error appeared. What can I do to resolve this error?

Thank you!

**Comment by u0078867** [comment]
I need more details on what you are doing exactly

**Comment by gabrielalsantos** [comment]
Hi,
I am trying to run my kinematics data again. But, that same error is showing up again when I intend to save my data. The following warning appears:

SAVING DATA BACK TO C3D: 
 Su:Processing subject C04 ...
   Se: Processing session Session 1 ...
     Tr: Processing trial RRC1.c3d ...>> 

Warning
Reference to non-existent field 'SUBJECTS'
Name: save ProcDataToC3D
Line: 89

I thought of the chance to be the difference between C3D files generated by Qualisys or by Vicon. Because when I ran the demo data, this error did not appear. Can this be? Or, can be another problem? 

Thank you!

Gabriela

**Comment by u0078867** [comment]
Hi Gabriela,

to really understand where the problem is, I need:
- the C3D files
- the exact settings for all the kinematics part

Thanks!

Davide

**Comment by gabrielalsantos** [comment]
Hi Davide,

I will send by email the C3d files. And, in relation about all the kinematics part:


First, we did a static trial for calibration with bilateral anatomic markers (ulnar styloid process, radial styloid process, medial epicondyle, lateral epicondyle, spinous process of C7, spinous process of T8, jugular notch, xiphoid process, inferior angle of the scapula, acromion angle, trig. the spine of the scapula) and cluster in trunk as well as bilateral cluster in scapula, arm and forearm. This trial is calsit in C3D files. 
After, I did functional trial (RF or LF) to determine the center of glenoumeral rotation. In this trial, I movement the patient’s upper arm (10 movements of circumduction). For this trial, I keep the clusters of the scapula, arm and trunk, and anatomic markers at scapula and trunk. 
Then, I performed three tasks trials (RRC1/2/3 or RS1/2/3). In this trial, the patient performed one reaching movement (the patient touches the object and returns to the starting position with hands on the thigh). For this, only cluster are maintained.

Thank you!

Gabriela
